DiscordBot.Commands/Helpers/VoiceChannelResolver.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Discord;
+using Discord.WebSocket;
+using DiscordBot.Commands.Core;
+using Microsoft.Extensions.Logging;
+
+namespace DiscordBot.Commands.Helpers
+{
+    public interface IVoiceChannelResolver
+    {
+        IVoiceChannel? GetVoiceChannel(SocketSlashCommand command);
+    }
+
+    public class VoiceChannelResolver : IVoiceChannelResolver
+    {
+        private readonly ILogger<VoiceChannelResolver> _logger;
+
+        public VoiceChannelResolver(ILogger<VoiceChannelResolver> logger)
+        {
+            _logger = logger;
+        }
+
+        public IVoiceChannel? GetVoiceChannel(SocketSlashCommand command)
+        {
+            var channel = command.GetOptionValue<IVoiceChannel>("channel");
```

Comment:
@Niewidzialny84 I wouldn't have hardcoded the option name in a helper like this. Preferably add it to function's parameters